plugins/chat/assets/javascripts/discourse/components/chat/thread-list.hbs

```diff
@@ -14,7 +16,9 @@
         {{loading-spinner size="medium"}}
       {{else}}
         {{#each this.sortedThreads as |thread|}}
-          <Chat::ThreadList::Item @thread={{thread}} />
+          {{#if (not thread.originalMessage.deletedAt)}}
```

Comment:
Again not sure if this sort of logic is supposed to go inside the component or outside...

Reply:
Can't we include this in sortedThread logic?

Reply:
Hmm yes good point will just put it there 👍